polus/plugins.py

```diff
@@ -850,15 +903,104 @@ def update_nist_plugins(gh_auth: typing.Optional[str] = None):
             _error_log(val_err, manifest, "update_nist_plugins")
 
 
-class Registry:
+class registry:
```

Comment:
Classes should be capitalized. We should also give it a more descriptive name like:
`WippPluginRegistry`